Makefile

```diff
@@ -43,4 +37,19 @@ docker-login:
 
 .PHONY: catalog
 channel-catalog: docker-login
-	@./hack/build_channel_catalog.sh
+	@BUNDLE_IMAGE="${BUNDLE_IMAGE}" \
```

Comment:
Personally, I don't like `@`. I want to see the CLI that was invoked -- especially in a case like this with all the variable settings, considering how significant they are to the process.

Reply:
Fine by me :)

Reply:
Fixed in 5a804b5